client/asset/btc/btc.go

```diff
@@ -1458,9 +1480,49 @@ func (btc *baseWallet) Balance() (*asset.Balance, error) {
 		Available: toSatoshi(balances.Mine.Trusted) - locked,
 		Immature:  toSatoshi(balances.Mine.Immature + balances.Mine.Untrusted),
 		Locked:    locked,
+		Other:     make(map[string]uint64),
 	}, nil
 }
 
+// Balance should return the total available funds in the wallet.
+func (btc *baseWallet) Balance() (*asset.Balance, error) {
+	bal, err := btc.balance()
+	if err != nil {
+		return nil, err
+	}
+
+	reserves := btc.reserves()
+	if reserves > bal.Available {
+		btc.log.Warnf("Available balance is below configured reserves: %f < %f",
+			toBTC(bal.Available), toBTC(reserves))
+		bal.Other["Reserves Deficit"] = reserves - bal.Available
+		reserves = bal.Available
+	}
+	bal.Other["Bond Reserves (locked)"] = reserves
```

Comment:
same as my comment above. I don't know if we do the same for dcr atm but I think it's best to have them declared somewhere.